types/lhr.d.ts

```diff
@@ -4,120 +4,116 @@
 
-    /**
-     * The full output of a Lighthouse run.
-     */
-    export interface Result {
-      /** The URL that was supplied to Lighthouse and initially navigated to. */
-      requestedUrl: string;
-      /** The post-redirects URL that Lighthouse loaded. */
-      finalUrl: string;
-      /** The ISO-8601 timestamp of when the results were generated. */
-      fetchTime: string;
-      /** The version of Lighthouse with which these results were generated. */
-      lighthouseVersion: string;
-      /** An object containing the results of the audits, keyed by the audits' `id` identifier. */
-      audits: Record<string, Audit.Result>;
-      /** The top-level categories, their overall scores, and member audits. */
-      categories: Record<string, Result.Category>;
-      /** Descriptions of the groups referenced by CategoryMembers. */
-      categoryGroups?: Record<string, Result.ReportGroup>;
+  /** The config settings used for these results. */
+  configSettings: Config.Settings;
+  /** List of top-level warnings for this Lighthouse run. */
+  runWarnings: string[];
+  /** A top-level error message that, if present, indicates a serious enough problem that this Lighthouse result may need to be discarded. */
+  runtimeError?: {code: string, message: string};
+  /** The User-Agent string of the browser used run Lighthouse for these results. */
+  userAgent: string;
+  /** Information about the environment in which Lighthouse was run. */
+  environment: Result.Environment;
+  /** Execution timings for the Lighthouse run */
+  timing: Result.Timing;
+  /** The record of all formatted string locations in the LHR and their corresponding source values. */
+  i18n: {rendererFormattedStrings: I18NRendererStrings, icuMessagePaths?: IcuMessagePaths};
+  /** An array containing the result of all stack packs. */
+  stackPacks?: Result.StackPack[];
+}
 
-      /** The config settings used for these results. */
-      configSettings: Config.Settings;
-      /** List of top-level warnings for this Lighthouse run. */
-      runWarnings: string[];
-      /** A top-level error message that, if present, indicates a serious enough problem that this Lighthouse result may need to be discarded. */
-      runtimeError?: {code: string, message: string};
-      /** The User-Agent string of the browser used run Lighthouse for these results. */
-      userAgent: string;
-      /** Information about the environment in which Lighthouse was run. */
-      environment: Environment;
-      /** Execution timings for the Lighthouse run */
-      timing: Result.Timing;
-      /** The record of all formatted string locations in the LHR and their corresponding source values. */
-      i18n: {rendererFormattedStrings: I18NRendererStrings, icuMessagePaths?: IcuMessagePaths};
-      /** An array containing the result of all stack packs. */
-      stackPacks?: Result.StackPack[];
-    }
+// Result namespace
+declare module Result {
+  interface Environment {
```

Comment:
this moves `LH.Environment` to `LH.Result.Environment`, but the only reference to this type by name is in this file (on `LH.Result['environment']`), so it doesn't really matter. It just felt weird to be out on top with all the other subtypes already nested under `LH.Result`.